packages/cardano-services/src/PgBoss/util.ts

```diff
@@ -0,0 +1,94 @@
+import { DataSource } from 'typeorm';
+import { Logger } from 'ts-log';
+import { Observable, firstValueFrom } from 'rxjs';
+import { Pool } from 'pg';
+import { PoolMetadataEntity, STAKE_POOL_METADATA_QUEUE, StakePoolMetadataTask } from '@cardano-sdk/projection-typeorm';
+import { contextLogger } from '@cardano-sdk/util';
+import { createHttpStakePoolMetadataService } from '../StakePool/HttpStakePoolMetadata/HttpStakePoolMetadataService';
+import PgBoss from 'pg-boss';
+
+const workerQueues = ['pool-metadata'] as const;
```

Comment:
This is the list of queues to be specified in the command line.
The association with `STAKE_POOL_METADATA_QUEUE` is in `queueHandlers`

Reply:
I understand, but both of those are representing a queue name. Why not use the same one?

Reply:
2b90ae0f0 should solve this

Reply:
Maybe let's rename the queue to from `"STAKE_POOL_METADATA"` to just `"pool-metadata"` or similar so that we have consistent cli options naming (this is similar to provider list when starting the provider server etc.)

Reply:
cf8c45b49 should solve this